client/components/hundred-year-loader-view.tsx

```diff
@@ -92,7 +94,15 @@ function VideoLoader( { isMobile, loadingText }: LoaderProps ) {
 					loop
 				/>
 			) }
-			<h1 className="wp-brand-font">{ loadingText }</h1>
+		</>
+	);
+
+	return hideVideoContainer ? (
+		videoElement
+	) : (
+		<FullPageVideoContainer isMobile={ isMobile }>
+			{ videoElement }
+			{ !! loadingText && <h1 className="wp-brand-font">{ loadingText }</h1> }
 		</FullPageVideoContainer>
```

Comment:
The `HundredYearLoaderView` component has been updated to support rendering video without a container so we can reuse it in different contexts.